### PR TITLE
js code to show the items in the cart #5

### PR DESCRIPTION
**Pull request by talaalsweiti**
js code to show the items in the cart when they are added by customer #5

### CODE REVIEW

File: Assets/js/script.js [hunk 1]
```diff
@@ -1 +1,109 @@
+let storedCarts = JSON.parse(localStorage.getItem("carts"));
+
+let intialCarts = [{
+    id: 0,
+    productName: "t-shirt",
+    productDetails: "bla bla",
```
Comment (Rahaf-96) [inline Assets/js/script.js]: don't forget to change `bla bla` 😆

File: Assets/js/script.js [hunk 2]
```diff
@@ -1 +1,109 @@
...
+
+        totalPrice += cartItem.productPrice;
+
+        allProducts.appendChild(productDiv);
+
+
+
+        //delete items
+        deleteButton.addEventListener("click", function () {
+          console.log(i);
```
Comment (Rahaf-96) [inline Assets/js/script.js]: remove `console.log(i)`

File: cart.html [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+<link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+  <title>Document</title>
```
Comment (Rahaf-96) [inline cart.html]: don't forget to change the page title

File: cart.html [hunk 4]
```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
```
Comment (Rahaf-96) [inline cart.html]: put in `href` the correct path for `style.css`.

File: cart.html [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+<link href="https://fonts.googleapis.com/css2?family=Quicksand&display=swap" rel="stylesheet">
+  <title>Document</title>
+</head>
+<body>
+  <div class="allProducts"> 
+</div>
+<script src="script.js"></script>
```
Comment (Rahaf-96) [inline cart.html]: same here, find where you put your `script.js` file, then change it's path.

**Review by Rahaf-96 — CHANGES_REQUESTED** [review]

File: Assets/css/cart.css [hunk 6]
```diff
@@ -0,0 +1,83 @@
+* {
+    margin: 0;
+    padding: 0;
+  }
+  
+  .allProducts{
+    display: flex;
+    justify-content: space-around;
+    /* border: 1px solid black; */
```
Comment (Rahaf-96) [inline Assets/css/cart.css]: delete this comment :)

File: Assets/js/cart.js [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+console.log(carts) ;  
+  total = carts.reduce(function(accumulator, currentValue) {
+    console.log(accumulator+ " acc");
+       return accumulator + currentValue.productPrice;
+   
+     },0)
+     
+     console.log(total + "total");
+     totalprice.textContent = "Total price : " + total;
```
Comment (tomduggan85) [inline Assets/js/cart.js]: I think you can move the price calculation and the totalPrice textContent setting outside of the forEach loop - you don't need to recalculate the total each time you create a new productDiv.

Comment (talaalsweiti) [reply Assets/js/cart.js]: thank you , done

Comment (tomduggan85) [reply Assets/js/cart.js]: Cool, thanks!

**Review by saharAdem — APPROVED** [review]

**Pull request merged**